Website/src/rendering/src/components/ShopCommon/QuantityInput.tsx

```diff
@@ -1,50 +1,85 @@
 import { PriceSchedule, RequiredDeep } from 'ordercloud-javascript-sdk';
-import { ChangeEvent, useState } from 'react';
+import { ChangeEvent, FocusEvent, useState } from 'react';
 
-interface ProductQuantityInputProps {
+interface QuantityInputProps {
+  controlId: string;
   priceSchedule: RequiredDeep<PriceSchedule>;
   label?: string;
   disabled?: boolean;
-  quantity: number;
+  quantity: number | string;
   onChange: (quantity: number) => void;
 }
 
-const ProductQuantityInput = ({
+const QuantityInput = ({
+  controlId,
   priceSchedule,
   disabled,
   quantity,
   onChange,
-}: ProductQuantityInputProps): JSX.Element => {
+}: QuantityInputProps): JSX.Element => {
   const [_quantity, setQuantity] = useState(quantity);
 
   const addDisabled =
     disabled || (priceSchedule.MaxQuantity ? _quantity >= priceSchedule.MaxQuantity : false);
   const subtractDisabled =
     disabled ||
-    (priceSchedule.MinQuantity ? _quantity < priceSchedule.MinQuantity : _quantity <= 0);
+    (priceSchedule.MinQuantity
+      ? _quantity <= priceSchedule.MinQuantity && _quantity <= 1
+      : _quantity <= 1);
+
+  const isInRange = (q: number) => {
```

Comment:
I find that `q` is not descriptive enough for a variable name. `quantity` would be better.